Tests/CarthageKitTests/VersionFileSpec.swift

```diff
@@ -1,7 +1,6 @@
 import Foundation
 import Quick
 import Nimble
-import Argo
```

Comment:
Woot! I imagine you're keeping the dependency cause other types still use it?

Reply:
Yes, Tentacle depends on Argo.